packages/documents/graphql/schema-types.js

```diff
@@ -50,9 +50,23 @@ type Meta {
   facebookTitle: String
   facebookImage: String
   facebookDescription: String
+  facebookGenerated: Boolean
+  facebookText: String
+  facebookFontSize: Int
+  facebookFontStyle: String
```

Comment:
(optional) Just to double check: what are the use cases for custom font styles?

Usually we infer from `meta.format.meta.kind` and `meta.kind`:
```
                  const kind =
                    (format && format.meta && format.meta.kind) ||
                    (meta && meta.kind)

                  const Headline =
                    metaHeadlines || kind === 'meta'
                      ? Meta.Headline
                      : kind === 'scribble'
                      ? Scribble.Headline
                      : Editorial.Headline
```
[Article template](https://github.com/orbiting/styleguide/blob/master/src/templates/Article/index.js#L288-L297)

Reply:
For meta share images, there are 2 different styles in use, as far as I know. We also support formatless share images, which don't have a predefined font style. Is this correct, @ovbm?

Reply:
Usecase for custom fontstyles are sharetiles like meta, where they could choose in the sketch file. 
![Screenshot 2021-04-14 at 10 38 24](https://user-images.githubusercontent.com/20746301/114680484-8b2c3d00-9d0d-11eb-8a38-d9f1d1d5b2da.jpg)

Reply:
BTW: Production confirms thomas: They only use meta font for meta....

Reply:
Why is Thomas always right? 😸 
So custom font style would only be for format-less docs. Is this a use case we want to support or shall we drop it and only show the generator when a format is present?

Reply:
🙈 

I would drop it. But keep the generator for documents without format just always with Rubis. Once we have concrete use cases for switching font styles for documents without formats we decide on how to infer it (e.g. from meta.kind or meta.template or a new prop).

Reply:
I would say show it for format-less docs.

Reply:
well all this gives me more work BUT it also simplifies things, which is good, I guess.

Reply:
🤪 sorry for that